lib/feature_requests.rb

```diff
@@ -0,0 +1,27 @@
+module Lita
+  # Returns cards that have been in Review column for more than two days
+  class FeatureRequests
+
+    SEVEN_DAYS = (60*60*168)
```

Comment:
Excellent point. Now that you mention it, I'm not entirely sure why I went with only half of the calculation anyway.